diagrams/components/lb/setup.ftl

```diff
@@ -1,17 +1,20 @@
 [#ftl]
 
-[#macro shared_lb_exec_diagram_config_overview occurrence ]
-    [@addComponentToOverview occurrence /]
+[#macro shared_lb_exec_diagram_config_solution occurrence ]
+    [@addComponentToSolutionDiagram occurrence /]
 
     [#list occurrence.Occurrences as subOccurrence ]
-        [#local solution = subOccurrence.Configuration.Solution ]
-        [#if (solution.IPAddressGroups![])?has_content ]
-            [@setupIPAddressGroupsRelationhsips
-                entityId=subOccurrence.Core.TypedId
-                IPAddressGroups=solution.IPAddressGroups
-                includeLocal=false
-                inbound=true
-            /]
+        [#if isOccurrenceInDiagram(subOccurrence)]
+            [#local solution = subOccurrence.Configuration.Solution ]
+            [#if (solution.IPAddressGroups![])?has_content ]
+                [@setupIPAddressGroupsRelationhsips
```

Comment:
This macro has a typo, but a search of the definition shows that it was defined that way. Should we correct that now?

Reply:
Fixed